test/WebJobs.Script.Tests/Workers/SharedMemoryManagerTests.cs

```diff
@@ -366,5 +366,22 @@ public async Task Dispose_VerifyAllSharedMemoryResourcesFreed()
             Assert.False(_mapAccessor.TryOpen(mapName1, out _));
             Assert.False(_mapAccessor.TryOpen(mapName1, out _));
         }
+
+        /// <summary>
+        /// Try to create two <see cref="SharedMemoryMap"/> with the same name.
+        /// Verify that the second attempt to create fails.
+        /// </summary>
+        [Fact]
+        public void CreateTwoNewSharedMemoryMapsWithSameName_VerifyFailure()
+        {
+            string mapName = Guid.NewGuid().ToString();
+            long contentSize = 100;
```

Comment:
We probably should test with the lengths that would give issues with the last approach.